### PR TITLE
Add LinearMouse

### PR DESCRIPTION
**Pull request by lujjjh**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/linearmouse.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+cask "linearmouse" do
+  version "0.3.1"
+  sha256 "c95286e1c6eb8592f57aac0c0dd31a157b93e7f0879cc62ed09e07b0638df5b0"
+
+  url "https://github.com/lujjjh/LinearMouse/releases/download/v#{version}/LinearMouse.dmg",
+      verified: "github.com/lujjjh/LinearMouse"
+  name "LinearMouse"
+  desc "For mouse users"
```
Comment (neutric) [inline Casks/linearmouse.rb:8]: This description says close to nothing. How about `desc "Customize mouse behavior"`?

File: Casks/linearmouse.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  homepage "https://linearmouse.org/"
+
+  livecheck do
+    url :url
+    strategy :github_latest
+  end
```
Comment (neutric) [inline Casks/linearmouse.rb:14]: `strategy :github_latest` comes at the cost of a limited API access and is only necessary if you also publish pre-releases. If all releases are production-ready, which it looks like to me, then you can omit the whole `livecheck` stanza and rely on the built-in `livecheck` for GitHub repos.

**Review by neutric — COMMENTED** [review]
Thanks for your contribution to `homebrew-cask`, @lujjjh!

**Comment by lujjjh** [comment]
@neutric Thank you for your kind review!

**Comment by bevanjkay** [comment]
Thank you @lujjjh

**Pull request merged**